cmd/run.go

```diff
@@ -572,3 +567,62 @@ func logVersion() {
 		"os/arch", fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	)
 }
+
+func forkIDIntervals(ctx context.Context, st *state.State, etherman *etherman.Client, genesisBlockNumber uint64) ([]state.ForkIDInterval, error) {
+	dbTx, err := st.BeginStateTransaction(ctx)
+	if err != nil {
+		return []state.ForkIDInterval{}, fmt.Errorf("error creating dbTx. Error: %v", err)
+	}
+	numberForkIDs, err := st.CountForkIDs(ctx, dbTx)
```

Comment:
We can put here the call to the function to get the forkids intervals:

`forkIDIntervals, err = st.GetForkIDs(ctx, dbTx)`

Then if len(forkIDIntervals)==0 we can execute the code to get it from L1, else we return the forkIDIntervals

Doing in this way we don't need to use CountForkIDs function and we are removing this query to the db